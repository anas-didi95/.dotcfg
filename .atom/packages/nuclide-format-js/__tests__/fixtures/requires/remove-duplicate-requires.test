
const {a} = require('Z');
const {b} = require('Z');

import type {A} from 'Z';
import type {B} from 'Z';

require('P');
require('P');

const M = require('M');
const M = require('M');

import type N from 'N';
import type N from 'N';

import typeof O from 'O';
import typeof O from 'O';

const [j] = require('Y');
const [j, k] = require('Y');

const x: A<B, N, O> = a(b, M, j, k);
