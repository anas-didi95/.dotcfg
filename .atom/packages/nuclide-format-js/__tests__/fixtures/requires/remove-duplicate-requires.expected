import type N from 'N';
import type {A, B} from 'Z';

import typeof O from 'O';

require('P');

const M = require('M');

const [j, k] = require('Y');
const {a, b} = require('Z');

const x: A<B, N, O> = a(b, M, j, k);
